### PR TITLE
Install anvil when dont found

### PR DESCRIPTION
**Pull request by sandhilt**
This pull request add anvil when dont found on system libraries.

### CODE REVIEW

File: DEV2DEV.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ OutputIndex uint64
 
 Input encoded by rollups-contract V2
 
-```
+```text
```
Comment (fabiooshiro) [inline DEV2DEV.md:17]: good!

**Review by fabiooshiro — APPROVED** [review]
LGTM

**Comment by fabiooshiro** [comment]
You will do the same for v1?

**Comment by fabiooshiro** [comment]
Are you planning to do the same for v1?

**Comment by sandhilt** [comment]
> Are you planning to do the same for v1?

For while, new features only for v2

**Review by fabiooshiro — APPROVED** [review]
LGTM!

**Pull request merged**